### PR TITLE
Revised cultural fit description

### PR DESCRIPTION
**Pull request by ralph-hawkins**
New wording by @superroz, to give a more precise definition and make it consistent with the way we describe 'Technical competence'.

### CODE REVIEW

File: VERSION.txt [hunk 1]
```diff
@@ -1 +1 @@
-2.3.1
+2.3.3
```
Comment (TheDoubleK) [inline VERSION.txt]: What happened to `2.3.2`?

**Comment by TheDoubleK** [comment]
Looks good.  Would be nice to fix the version number, but other than that  :+1:

**Comment by TheDoubleK** [comment]
You'll need to add a small story to the backlog to pull this updated content into the buyer app.

**Comment by pcraig3** [comment]
Also you'll need to add a small story to un-fuck everything up.

**Pull request merged**